shuttle/tests/advanced/abort_freedom.rs

```diff
@@ -0,0 +1,93 @@
+use shuttle::scheduler::DfsScheduler;
+use shuttle::sync::Mutex;
+use shuttle::{Config, Runner};
+use std::collections::VecDeque;
+use std::sync::Arc;
+
+// check_dfs with immediately_return_on_panic set to true
+pub fn check_dfs<F>(f: F)
```

Comment:
Why is this `pub`?  And why overload `check_dfs`?

Actually, why even define a function? It's only used once here?

Reply:
Copy-pasted it over, then didn't think to clean it up. I agree it's better to inline the function.